freeze/pyzo_freeze.py

```diff
@@ -74,9 +73,23 @@ def _find_modules(root, extensions, skip, parent=""):
                     yield moduleName
 
 
+def get_python_stdlib_path():
```

Comment:
So if I understand correctly, the code below is copied (more or less) from `distutils`? It feels kind of fragile to have this triage here like that. Knowing what the original source was gives some reassurance 😄 so let's add a comment if that's indeed the case.
```suggestion
def get_python_stdlib_path():
    # This code is extracted from distutils.sysconfig.get_python_lib (distutils is deprecated in 3.12)
```

Reply:
The original implementation can be seen here:
https://github.com/python/cpython/blob/a673248d6cb1ae5d5b7fbf0ab6c8db8c32df0eee/Lib/distutils/sysconfig.py#L304-L346